.github/workflows/docs-checks.yml

```diff
@@ -1,109 +0,0 @@
-name: Check Docs
-# https://github.com/marketplace/actions/sphinx-build
-
-on:  # Trigger the workflow on push or pull request, but only for the master branch
-  push:
-    branches: [master, "release/*"]
-  pull_request:
-    branches: [master, "release/*"]
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
-  cancel-in-progress: ${{ ! (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/')) }}
-
-jobs:
-  doctest:
```

Comment:
Removing `doctest` since we do have our test at https://github.com/Lightning-AI/lightning/blob/61c28cb428a13c2aea6d7f3f55e0f00431a4ea4e/.github/workflows/ci-pytorch_test-full.yml#L95

Reply:
it is very different and just name colisiotn, this doctest is running python code in `.rst` files, compare to doctest you refer is running examples un docstring in `.py` files

Reply:
Resolving this comment as the removal of doctest has been reverted.